run_model-indep_with-ml-cats_limits.sh

```diff
@@ -0,0 +1,142 @@
+        --variable mt_tot_puppi \
+        --sm-gg-fractions ${CMSSW_BASE}/src/CombineHarvester/MSSMvsSMRun2Legacy/data/higgs_pt_reweighting_fullRun2_v2.root \
+        --parallel 10 2>&1 | tee -a ${defaultdir}/logs/morph_mssm_log.txt
+
+    ############
+    # combining outputs
+    ############
+    mkdir -p ${datacarddir}/combined/cmb/
+    rsync -av --progress ${datacarddir}/201?/htt_*/* ${datacarddir}/combined/cmb/ 2>&1 | tee -a ${defaultdir}/logs/copy_datacards.txt
+
+elif [[ $MODE == "ws" ]]; then
+    ############
+    # workspace creation
+    ############
+
+    combineTool.py -M T2W -o "ws.root" \
+    -P HiggsAnalysis.CombinedLimit.PhysicsModel:multiSignalModel \
+    --PO '"map=^.*/ggh_(i|t|b).?$:r_ggH[0,0,200]"' \
+    --PO '"map=^.*/bbh$:r_bbH[0,0,200]"' \
+    -i ${datacarddir}/combined/cmb/ \
+    -m 125.0 --parallel 4 | tee -a ${defaultdir}/logs/workspace_independent.txt
+
+    ############
+    # job setup creation
+    ############
+    cd ${defaultdir}/limits_ind/condor
+    combineTool.py -m "60,80,100,120,125,130,140,160,180,200,250,300,350,400,450,500,600,700,800,900,1000,1200,1400,1600,1800,2000,2300,2600,2900,3200,3500" \
+    -M AsymptoticLimits \
+    --rAbsAcc 0 \
+    --rRelAcc 0.0005 \
+    --boundlist ${CMSSW_BASE}/src/CombineHarvester/MSSMvsSMRun2Legacy/input/mssm_boundaries.json \
+    --setParameters r_ggH=0,r_bbH=0 \
+    --redefineSignalPOIs r_bbH \
+    -d ${datacarddir}/combined/cmb/ws.root \
+    --there -n ".bbH" \
+    --job-mode condor \
+    --dry-run \
+    --task-name bbH_full_cmb \
+    --X-rtd MINIMIZER_analytic \
+    --cminDefaultMinimizerStrategy 0 \
+    --cminDefaultMinimizerTolerance 0.01 \
+    -v 1 | tee -a ${defaultdir}/logs/job_setup_modelind_bbh.txt
+
+    combineTool.py -m "60,80,100,120,125,130,140,160,180,200,250,300,350,400,450,500,600,700,800,900,1000,1200,1400,1600,1800,2000,2300,2600,2900,3200,3500" \
+    -M AsymptoticLimits \
+    --rAbsAcc 0 \
+    --rRelAcc 0.0005 \
+    --boundlist ${CMSSW_BASE}/src/CombineHarvester/MSSMvsSMRun2Legacy/input/mssm_boundaries.json \
+    --setParameters r_ggH=0,r_bbH=0 \
+    --redefineSignalPOIs r_ggH \
+    -d ${datacarddir}/combined/cmb/ws.root \
+    --there -n ".ggH" \
+    --job-mode condor \
+    --dry-run \
+    --task-name ggH_full_cmb \
+    --X-rtd MINIMIZER_analytic \
+    --cminDefaultMinimizerStrategy 0 \
+    --cminDefaultMinimizerTolerance 0.01 \
+    -v 1 | tee -a ${defaultdir}/logs/job_setup_modelind_ggh.txt
+
+elif [[ $MODE == "submit" ]]; then
+    ############
+    # job submission
+    ############
+    cd ${defaultdir}/limits_ind/condor
+    condor_submit condor_bbH_full_cmb.sub
+    condor_submit condor_ggH_full_cmb.sub
+
+    elif [[ $MODE == "submit-local" ]]; then
+    ############
+    # job submission
+    ############
+    cp scripts/run_limits_locally.py ${defaultdir}/limits_ind/condor
+    cd ${defaultdir}/limits_ind/condor
+    python run_limits_locally.py --cores 10 --njobs 27 --taskname condor_bbH_full_cmb.sh
```

Comment:
With the full mass range, we have 31 jobs. The same holds for the classic model-independent case but there I could not comment as there was no change.
```suggestion
    python run_limits_locally.py --cores 10 --njobs 31 --taskname condor_bbH_full_cmb.sh
```